magento/pwa-studio
# [doc] TargetableModule.spliceSource() examples and debugging instructions

## Description

Adds examples for using the `TargetableModule.spliceSource()` function as well as debugging instructions.

## Related Issue





Closes PWA-1710 and Closes #3155 

## Acceptance

Any core developer
@Jordaneisenburger (optional)

### Verification Stakeholders

Any core developer
@Jordaneisenburger (optional)

### Specification



### Verification Steps

1. Navigate to the docs directory: `cd pwa-devdocs`
2. Run the HTML preview server: `yarn develop`
3. Navigate to the TargetableModule docs: `/pwa-buildpack/reference/targetables/TargetableModule/`
4. Verify examples make sense and accurate
5. Verify debugging steps are correct and make sense

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
